OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/preferencecenter/PreferenceCenterTopic.java

```diff
@@ -0,0 +1,33 @@
+package com.optimove.android.preferencecenter;
+
+import java.util.List;
+
+public class PreferenceCenterTopic {
```

Comment:
now since everything moved under a common PC namespace, can call it just Topic. Sorry :)